Makefile

```diff
@@ -75,11 +75,12 @@ libbpfgo-static: $(VMLINUXH) | $(LIBBPF_OBJ)
 		.
 
 libbpfgo-static-test: libbpfgo-test-bpf-static
-	CC=$(CLANG) \
+	sudo env PATH=$(PATH) \
+		CC=$(CLANG) \
 		CGO_CFLAGS=$(CGO_CFLAGS_STATIC) \
 		CGO_LDFLAGS=$(CGO_LDFLAGS_STATIC) \
 		GOOS=linux GOARCH=$(ARCH) \
-		sudo -E -- go test \
+		go test \
 		-tags netgo -ldflags $(CGO_EXTLDFLAGS_STATIC) \
```

Comment:
can you add `-v` in here while you're at it?

Reply:
Done.